src/ffi/types.rs

```diff
@@ -267,6 +270,58 @@ impl std::fmt::Display for Offer {
 	}
 }
 
+pub struct HumanReadableName {
+	pub(crate) inner: LdkHumanReadableName,
+}
+
+impl HumanReadableName {
+	pub fn into_inner(&self) -> LdkHumanReadableName {
+		self.inner.clone()
+	}
+
+	pub fn from_encoded(encoded: &str) -> Result<Self, Error> {
```

Comment:
Please provide docs on all public methods. Feel free to copy them over from the LDK counterpart.